instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/context/ContextPropagationDebug.java

```diff
@@ -22,9 +22,9 @@
       ContextKey.named("thread-propagation-locations");
 
   private static final boolean THREAD_PROPAGATION_DEBUGGER =
-      Boolean.getBoolean("otel.threadPropagationDebugger");
+      Boolean.getBoolean("otel.javaagent.experimental.thread-propagation-debugger.enabled");
```

Comment:
What do you think about `otel.javaagent.debug.thread-propagation.enabled` instead? We could also set its default value to `Config.isAgentDebugEnabled()`